<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prestashop &amp; Colibri</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/png/resources/burn.png</normaloff>:/png/resources/burn.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Automation</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Pending Orders</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Processed Order</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="lcdPending">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>121</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Filled</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="lcdProcessed">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>121</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu</family>
              <pointsize>26</pointsize>
             </font>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Filled</enum>
            </property>
            <property name="intValue" stdset="0">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QLabel" name="errors">
            <property name="font">
             <font>
              <pointsize>40</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(85, 170, 0);</string>
            </property>
            <property name="text">
             <string>No Errors</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>200</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string>Full auto now</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Manual</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="0">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Print autocolantes to A4</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/document-print.png</normaloff>:/png/resources/document-print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>Print registos de envio</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/document-print.png</normaloff>:/png/resources/document-print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Fetch Orders</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/copy doc.png</normaloff>:/png/resources/copy doc.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLCDNumber" name="lcdOrdersToProcess"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Fetched Orders</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>Print Facturas</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/document-print.png</normaloff>:/png/resources/document-print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="autoA4Offset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Print autocolantes to Brother</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/document-print.png</normaloff>:/png/resources/document-print.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="10" column="0" colspan="3">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Set Tracking Numbers</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/internet-mail.png</normaloff>:/png/resources/internet-mail.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="log"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Saidas</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>131</height>
           </size>
          </property>
          <property name="text">
           <string>Registar Saidas</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/forward all.png</normaloff>:/png/resources/forward all.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Tools</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>131</height>
           </size>
          </property>
          <property name="text">
           <string>Check tracking status</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/forward all.png</normaloff>:/png/resources/forward all.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="addByOrderID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add by order ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addByOrderIDAct">
          <property name="toolTip">
           <string>Add by order ID</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/plus.png</normaloff>:/png/resources/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addByCustomerID">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add by customer ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addByCustomerIDAct">
          <property name="toolTip">
           <string>Add by customer ID</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/plus.png</normaloff>:/png/resources/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addByCustomerName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add by customer name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="addByCustomerNameAct">
          <property name="toolTip">
           <string>Add by customer name</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/plus.png</normaloff>:/png/resources/plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="deleteAllAct">
          <property name="toolTip">
           <string>delete</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/exclamation.png</normaloff>:/png/resources/exclamation.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="deleteAct">
          <property name="toolTip">
           <string>delete</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/png/resources/stop.png</normaloff>:/png/resources/stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Nome</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Morada</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pais</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Valor</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Order ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Tracking</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <column>
           <property name="text">
            <string>Produto</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantidade</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionImprimir_minha_morada_para_A4"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionImprimir_minha_morada_para_A4">
   <property name="text">
    <string>Imprimir minha morada para A4</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
